react/Header/Navigation/Navigation.js

```diff
@@ -59,11 +60,20 @@ export default function Navigation({ locale, linkRenderer, activeTab, divider })
               <li className={styles.item} key={i}>
                 {
                   linkRenderer({
-                    children: name,
+                    children: [
+                      name,
+                      name === newBadgeTab && (
+                        <NewBadge
+                          key={name}
+                          className={styles.newBadge}
+                        />
+                      )
+                    ],
                     'data-analytics': analytics,
                     className: classnames({
                       [styles.link]: true,
-                      [styles.link_isActive]: name === activeTab
+                      [styles.link_isActive]: name === activeTab,
+                      [styles.link_smallText]: locale === 'AU' && newBadgeTab === 'Company Reviews'
```

Comment:
I understand why we are doing this but it still doesnt feel right, are we happy to drop the font size of the whole primary nav to accomodate a new badge? 🤔

Reply:
I tried uncommenting this line, and it seemed fine to me. The new badge is placed on top of the tab when the screen is between 740px and 960px wide, so it doesn't run out of room.